package.json

```diff
@@ -19,7 +19,7 @@
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
I think you should remove the --openssl-legacy-provider part

Reply:
true, fixed!